### PR TITLE
Add components sample for CloudFormation

### PR DESCRIPTION
**Pull request by nouvionp**
Adding Components sample for CloudFormation

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: service-templates/load-balanced-fargate-svc/v1/instance_infrastructure/cloudformation.yaml [hunk 1]
```diff
@@ -110,6 +110,41 @@ Resources:
               - Effect: 'Allow'
                 Action: 'sns:Publish'
                 Resource: '{{environment.outputs.SNSTopicArn}}'
+      PermissionsBoundary: !Ref TaskRolePermissionBoundaryPolicy
+      ManagedPolicyArns:
+        - !Ref BaseTaskRoleManagedPolicy
+        {{ service_instance.components.custom_runtime.outputs
```
Comment (mikewrighton) [inline service-templates/load-balanced-fargate-svc/v1/instance_infrastructure/cloudformation.yaml:116]: I think component outputs are now referenced with `service_instance.components.default.outputs`?

File: service-templates/load-balanced-fargate-svc/README.md [hunk 2]
```diff
@@ -31,6 +31,12 @@ The template also provisions a CodePipeline based pipeline to pull your applicat
 3. unit_test_command: The command to run to unit test the application code
 4. environment_account_ids: The environment account ids for service instances using cross account environment
 
+### (Optional) Extend the Service with a Component
+
+The service template which you used above will provision a Fargate service with the infrastructure required for networking, monitoring and scaling the service. In practice you may want to use a template like this for many different purposes and can use components to extend the service template to meet your requirements.
+
+Jinja filters will be used to optionally incorporate component outputs of our s3-bucket-component sample which provisions an S3 bucket and defines an IAM Policy for accessing that bucket will be associated with the service instance.
+
```
Comment (mikewrighton) [inline service-templates/load-balanced-fargate-svc/README.md:39]: This paragraph doesn't read cleanly, to me. For the Terraform template I added this:

```
The s3-bucket-component sample provisions an S3 bucket and defines an IAM Policy for accessing it. If present, the bucket policy and bucket ARN outputs from the s3-bucket-component will be used in the ECS task role and ECS environment variables, respectively.
```

File: service-templates/load-balanced-fargate-svc/v1/instance_infrastructure/cloudformation.yaml (hunk 1 above)
Comment (nouvionp) [reply service-templates/load-balanced-fargate-svc/v1/instance_infrastructure/cloudformation.yaml:116]: oh yes you are correct, good catch, thanks!

File: service-templates/load-balanced-fargate-svc/README.md (hunk 2 above)
Comment (nouvionp) [reply service-templates/load-balanced-fargate-svc/README.md]: Updated

File: service-templates/load-balanced-fargate-svc/v1/instance_infrastructure/cloudformation.yaml (hunk 1 above)
Comment (nouvionp) [reply service-templates/load-balanced-fargate-svc/v1/instance_infrastructure/cloudformation.yaml]: fixed

**Comment by nouvionp** [comment]
Got offline approval.

**Pull request closed without merging**